components/colors/colors.scss

```diff
@@ -10,83 +10,77 @@ $d2l-color-tungsten: #6e7376 !default;
 $d2l-color-ferrite: #494c4e !default;
 
 // zircon
-$d2l-color-zircon-plus-2: #bbf1fa !default;
-$d2l-color-zircon-plus-1: #00bddd !default;
-$d2l-color-zircon: #00849c !default;
-$d2l-color-zircon-minus-1: #005b6b !default;
+$d2l-color-zircon-plus-2: #e0feff !default;
+$d2l-color-zircon-plus-1: #00d2ed !default;
+$d2l-color-zircon: #008eab !default;
+$d2l-color-zircon-minus-1: #035670 !default;
 
 // celestine
-$d2l-color-celestine-plus-2: #e8f2f9 !default;
+$d2l-color-celestine-plus-2: #e8f8ff !default;
 $d2l-color-celestine-plus-1: #29a6ff !default;
 $d2l-color-celestine: #006fbf !default;
-$d2l-color-celestine-minus-1: #005694 !default;
+$d2l-color-celestine-minus-1: #004489 !default;
 
 // amethyst
-$d2l-color-amethyst-plus-2: #e9e6ff !default;
-$d2l-color-amethyst-plus-1: #afa1ff !default;
-$d2l-color-amethyst: #7862f0 !default;
-$d2l-color-amethyst-minus-1: #4c3f99 !default;
+$d2l-color-amethyst-plus-2: #f2f0ff !default;
+$d2l-color-amethyst-plus-1: #8982ff !default;
+$d2l-color-amethyst: #6038ff !default;
+$d2l-color-amethyst-minus-1: #4500db !default;
 
 // fluorite
-$d2l-color-fluorite-plus-2: #f6e1fa !default;
-$d2l-color-fluorite-plus-1: #ea82ff !default;
-$d2l-color-fluorite: #b24ac7 !default;
-$d2l-color-fluorite-minus-1: #7e358c !default;
+$d2l-color-fluorite-plus-2: #f9ebff !default;
+$d2l-color-fluorite-plus-1: #ce68fa !default;
+$d2l-color-fluorite: #9d1fd4 !default;
+$d2l-color-fluorite-minus-1: #6900a0 !default;
 
 // tourmaline
-$d2l-color-tourmaline-plus-2: #fae1ed !default;
-$d2l-color-tourmaline-plus-1: #ff389b !default;
-$d2l-color-tourmaline: #bf2a75 !default;
-$d2l-color-tourmaline-minus-1: #96215c !default;
+$d2l-color-tourmaline-plus-2: #ffebf6 !default;
+$d2l-color-tourmaline-plus-1: #fd4e9d !default;
+$d2l-color-tourmaline: #d40067 !default;
+$d2l-color-tourmaline-minus-1: #990056 !default;
 
 // cinnabar
-$d2l-color-cinnabar-plus-2: #fae1e2 !default;
-$d2l-color-cinnabar-plus-1: #ff6b70 !default;
+$d2l-color-cinnabar-plus-2: #ffede8 !default;
+$d2l-color-cinnabar-plus-1: #ff575a !default;
 $d2l-color-cinnabar: #cd2026 !default;
-$d2l-color-cinnabar-minus-1: #a1191d !default;
+$d2l-color-cinnabar-minus-1: #990006 !default;
 
 // carnelian
-$d2l-color-carnelian-plus-1: #fae3cf !default;
+$d2l-color-carnelian-plus-1: #fff3e0 !default;
 $d2l-color-carnelian: #e87511 !default;
-$d2l-color-carnelian-minus-1: #a6540d !default;
-$d2l-color-carnelian-minus-2: #804008 !default;
+$d2l-color-carnelian-minus-1: #ba4700 !default;
+$d2l-color-carnelian-minus-2: #7d2600 !default;
 
 // citrine
-$d2l-color-citrine-plus-1: #fae5c8 !default;
+$d2l-color-citrine-plus-1: #fff9d6 !default;
 $d2l-color-citrine: #ffba59 !default;
-$d2l-color-citrine-minus-1: #a87b3b !default;
-$d2l-color-citrine-minus-2: #694b25 !default;
-
-// topaz
-$d2l-color-topaz-plus-1: #fffbb2 !default;
-$d2l-color-topaz: #f5ec5a !default;
-$d2l-color-topaz-minus-1: #a69f21 !default;
-$d2l-color-topaz-minus-2: #635f15 !default;
```

Comment:
So the old colors repo is [referencing topaz a couple times](https://github.com/BrightspaceUI/colors/blob/2ab6adb98e3de8fd8ba9e27d6f46526f1e392aea/d2l-colors.scss#L12).

We can either: 
1. Investigate whether those references are actually used and if not, remove them as well
2. Hardcode the HEX for topaz in there

Reply:
Good catch, I'll investigate.

Reply:
There don't seem to be any references to lurite or chartronic so I'll remove those.

Reply:
Sounds good!